scripts/start-encrypted-cluster.sh

```diff
@@ -0,0 +1,36 @@
+# creates a encrypted cluster (sharded on 8.0 server)
+
+export CWD=$(pwd);
+mkdir encrypted-cluster
+cd encrypted-cluster
+
+# note: 
+  # we're using drivers-evergreen-tools which is a repo that handles cluster set-up for us. 
```

Comment:
```suggestion
  # we're using drivers-evergreen-tools which is a repo used by MongoDB drivers to start clusters for testing.  
  # Note: drivers-evergreen-tools is NOT intended to start and manage production servers.
```

we should be explicit that this is only intended to be used for testing

Reply:
Since we're not in a mongodb driver right now, but in mongoose, I'd rather not just say drivers-evergreen-tools is just 'used by mongodb drivers'.

Reply:
I don't feel strongly about whether or not we mention drivers specifically, but we need to emphasize that this is only intended to testing and not an officially supported product for managing clusters.

Reply:
might be easier to just leave 'used by drivers' to make that point clear!